src/containers/Tenant/Diagnostics/Overview/AsyncReplicationInfo/AsyncReplicationInfo.tsx

```diff
@@ -0,0 +1,85 @@
+import {Flex, Text} from '@gravity-ui/uikit';
+
+import {InfoViewer} from '../../../../../components/InfoViewer';
+import type {TEvDescribeSchemeResult} from '../../../../../types/api/schema';
+import {useTypedSelector} from '../../../../../utils/hooks';
+import {AsyncReplicationState} from '../../../../AsyncReplicationState';
+import {getEntityName} from '../../../utils';
+import {AsyncReplicationPaths} from '../AsyncReplicationPaths';
+
+import {Credentials} from './Credentials';
+
+interface AsyncReplicationProps {
+    data?: TEvDescribeSchemeResult;
+}
+
+/** Displays overview for Replication EPathType */
+export function AsyncReplicationInfo({data}: AsyncReplicationProps) {
+    const entityName = getEntityName(data?.PathDescription);
+
+    const {error: schemaError} = useTypedSelector((state) => state.schema);
+
+    if (schemaError) {
+        return <div className="error">{schemaError.statusText}</div>;
+    }
+
+    if (!data) {
+        return <div className="error">No {entityName} data</div>;
+    }
+
+    return (
+        <Flex direction="column" gap="4">
+            <InfoViewer
+                title={entityName}
+                info={[
+                    {
+                        label: 'Name',
+                        value: data.PathDescription?.ReplicationDescription?.Name,
```

Comment:
Currently we aren't displaing entities names in `Overview`, since they are present in schema tree. Should we do this here?

Reply:
Removed.